src/components/Nav/index.js

```diff
@@ -279,7 +279,17 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
     },
     {
       text: "community",
-      to: "/community/",
+      ariaLabel: "community-menu",
```

Comment:
We'll have to discuss as a team how we want to handle the community section as a whole. It might not make sense right now to just have rollup grants as the only subpage in this dropdown.

Reply:
Agreed - I think this creates a slippery slope in terms of what deserves to be in the primary Community dropdown. Until we do some thinking around that, I'd suggest we just link to this page from the Community page for now (e.g. in the "Upcoming events" and/or "How can I get involved?" sections.

Reply:
We could also include a link to this page from the rollups developer docs: https://ethereum.org/en/developers/docs/layer-2-scaling/#rollups

Reply:
One thing I want to highlight: there are still some minor updates that need to be made on the index.md file. As you have seen, links to be included, dates to be updates etc.

I made this PR regardeless so that the page is already "in there" and can be made live with another small PR once I have that info.

I think the link to the developer docs should only be made once its really going live, right?

Reply:
> I think the link to the developer docs should only be made once its really going live, right?

I don't see any harm in including the links now @ETHorHIL since those pages are live.

Reply:
Ok